newsroom/push/utils.py

```diff
@@ -31,14 +31,16 @@ def set_dates(doc: dict[str, Any]):
     doc.setdefault(VERSION, 1)
 
 
-def fix_updates(doc: dict[str, Any], next_item: Item):
-    service = get_resource_service("content_api")
+async def fix_updates(doc: dict[str, Any], next_item: WireItem):
+    wire_search = WireSearchServiceAsync()
     ancestors = (doc.get("ancestors") or []) + [doc["guid"]]
 
     for i in range(50):
-        updates = {"ancestors": ancestors + (next_item.get("ancestors") or []), "original_id": doc["original_id"]}
-        service.system_update(next_item["_id"], updates, next_item)
-        next_item = service.find_one(req=None, evolvedfrom=next_item["_id"])
+        updates = {"ancestors": ancestors + (next_item.ancestors or []), "original_id": doc["original_id"]}
+        # updates = {"ancestors": ancestors + (next_item.get("ancestors") or []), "original_id": doc["original_id"]}
+        await wire_search.service.system_update(next_item.id, updates)
+        # service.system_update(next_item.id, updates, next_item)
```

Comment:
we can remove these comments I think
```suggestion
        updates = {"ancestors": ancestors + (next_item.ancestors or []), "original_id": doc["original_id"]}
        await wire_search.service.system_update(next_item.id, updates)
```